app/apis/hackney_api/repairs_client.rb

```diff
@@ -168,16 +168,17 @@ def get_property_by_postcode(postcode, min_level=LEVEL_NON_DWELL, max_level=LEVE
       )
     end
 
-    def get_property_by_address(address, min_level=LEVEL_NON_DWELL, max_level=LEVEL_ESTATE)
+    def get_property_by_address(address, min_level=LEVEL_NON_DWELL, max_level=LEVEL_ESTATE, limit: nil)
```

Comment:
is there a reason why we need a `nil` default? We could just make it mandatory. And get rid of the compact here and in the test stub.

Reply:
well I thought that the `limit` parameter was kinda optional thing. If you pass `nil`, the API call won't have the parameter and then you'll get the default behaviour (which seems to be 100 results).

Reply:
I was thinking that since we never use the `nil` default we should just remove the code for it.

Reply:
done

Reply:
what do you think of this last patch?